server/cluster/metadata/topology_manager.go

```diff
@@ -231,14 +248,15 @@ func (m *TopologyManagerImpl) RemoveTable(ctx context.Context, shardID storage.S
 	}
 
 	// Update shardView in storage.
+	newShardView := storage.ShardView{
+		ShardID:   shardView.ShardID,
+		Version:   prevVersion + 1,
+		TableIDs:  newTableIDs,
+		CreatedAt: uint64(time.Now().UnixMilli()),
+	}
```

Comment:
How about declaring a new method for ShardView to avoid calling `time.Now().UnixMilli()` again and again?

Reply:
Now, you have to call `generateCreateTime` again and again. Why not define a method called `NewShardView` in the storage package and generate the create timestamp there?